libs/src/services/solana/SolanaWeb3Manager.ts

```diff
@@ -58,6 +58,12 @@ type CreateSenderParams = Omit<
   | 'identityService'
 > & { feePayerOverride: Nullable<string> }
 
+type MintName = 'usdc' | 'audio'
+type KeyAndAddress = {
+  address: string
```

Comment:
I suspect this is a legacy thing, but why do we need both address and pubkey? Could we not just call `toString` on pubkey or something?

Reply:
excellent point, I'm going to change this